### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
### Description

This pull request is to address misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/jdaviz/commit/c6ade9eec0701a1a6829f9c0704172b158ca1834#commitcomment-62588527

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/jdaviz/commit/15ff520977168406eae214973fb1ee7ab8e051ba

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

Fixes #<Issue Number>

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/app.vue [hunk 1]
```diff
@@ -204,7 +204,7 @@ div.output_wrapper {
 }
 
 .v-toolbar__items .v-btn {
-  /* allow voolbar-items styling to pass through tooltip wrapping span */
+  /* allow toolbar-items styling to pass through tooltip wrapping span */
```
Comment (jsoref) [inline jdaviz/app.vue:207]: It's possibly this should be `v-toolbar` ...

File: jdaviz/configs/specviz/helper.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def _set_scale(self, scale, axis, min_val=None, max_val=None):
             elif isinstance(min_val, u.Quantity):
                 # Convert user's value to axis' units
                 min_val = min_val.to(axis.unit).value
-            # If auto, set to min_valimum wavelength value
+            # If auto, set to min axis wavelength value
```
Comment (jsoref) [inline jdaviz/configs/specviz/helper.py:147]: This is a significant change. I couldn't figure out what a `valimum` was and decided that it was a corruption of minimum/maximum. I've tried to adjust these comments to match what the code appears to be doing.

File: jdaviz/data/linelists/SDSS.csv [hunk 3]
```diff
@@ -35,14 +35,14 @@ Hα,6564.61,Emission,8.,8.
...
-CaII,8544.44,Absorptio,-1.,0.
-CaII,8664.52,Absorptio,-1.,0.
+K,3934.77,Absorption,-1.,0.
+H,3969.58,Absorption,-1.,0.
+G,4305.61,Absorption,-1.,0.
+Mg,5176.7,Absorption,-1.,0.
+Na,5895.6,Absorption,-1.,0.
+CaII,8500.36,Absorption,-1.,0.
+CaII,8544.44,Absorption,-1.,0.
+CaII,8664.52,Absorption,-1.,0.
```
Comment (jsoref) [inline jdaviz/data/linelists/SDSS.csv:45]: If this is data generated by an external module, then this change is wrong.

File: jdaviz/data/linelists/linelist_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "Common Stellar": {
     "units": "Angstrom",
     "filename_base": "Common_stellar",
-    "Citations/Notes": " Common stellar lines.\n\n Copyrigtht (C) 1999-2004 by Christian Buil\n http://www.astrosurf.com/buil/us/spe2/hresol5.htm"
+    "Citations/Notes": " Common stellar lines.\n\n Copyright (C) 1999-2004 by Christian Buil\n http://www.astrosurf.com/buil/us/spe2/hresol5.htm"
```
Comment (jsoref) [inline jdaviz/data/linelists/linelist_metadata.json:5]: Changing copyright statements tends to scare lawyers.

File: notebooks/concepts/Specviz from splot concept notebook.ipynb [hunk 5]
```diff
@@ -1127,7 +1127,7 @@
...
     "\n",
-    "    Note, though: two piece of arithmetic are critical: a way to subtract a fitted continuum and a way to divide by it.  These could be implemented as part of the *fitting* plugin, however, to avoid having to write a separate arithmetic plugin for this purpose. So the baseline specviz should anclude options for subtracting/dividing by continuum in the fitting UI, and a more general arithmetic plugin (which *must* use the same underlying `specutils` machinery to not confuse users) is an option to consider later."
+    "    Note, though: two piece of arithmetic are critical: a way to subtract a fitted continuum and a way to divide by it.  These could be implemented as part of the *fitting* plugin, however, to avoid having to write a separate arithmetic plugin for this purpose. So the baseline specviz should include options for subtracting/dividing by continuum in the fitting UI, and a more general arithmetic plugin (which *must* use the same underlying `specutils` machinery to not confuse users) is an option to consider later."
```
Comment (jsoref) [inline notebooks/concepts/Specviz from splot concept notebook.ipynb:1130]:
```diff
-anclude
+include
```

File: jdaviz/app.vue (hunk 1 above)
Comment (pllim) [inline jdaviz/app.vue:207]: Yes, I think it is possible indeed.

```suggestion
  /* allow v-toolbar-items styling to pass through tooltip wrapping span */
```

File: jdaviz/configs/specviz/helper.py (hunk 2 above)
Comment (pllim) [inline jdaviz/configs/specviz/helper.py:147]: I think someone tried to replace "minimum" with "min_val" but gave up half-way...

```suggestion
            # If auto, set to min_val wavelength value
```

Comment (pllim) [reply jdaviz/configs/specviz/helper.py:147]: But your change makes more sense...

File: jdaviz/data/linelists/SDSS.csv (hunk 3 above)
Comment (pllim) [reply jdaviz/data/linelists/SDSS.csv:45]: Indeed, might be some char limit in the parser logic or something. Let's wait for inputs from other devs.

**Review by pllim — COMMENTED** [review]
Thanks. Good catch! Though I will withhold from approving until some outstanding questions are answered by other devs.

File: jdaviz/data/linelists/SDSS.csv (hunk 3 above)
Comment (rosteen) [reply jdaviz/data/linelists/SDSS.csv:45]: This change is fine, I don't think we use the field anyway and the file is unlikely to ever be refreshed.

**Review by rosteen — APPROVED** [review]
Looks good to me, thanks!

**Comment by rosteen** [comment]
I believe the test failure was fixed on `main`, if you rebase I think the CI will pass.

**Comment by jsoref** [comment]
OK, CI wants a changelog entry or probably some thing saying I don't need one. Do I need one?

(Also, I'll be effectively unavailable for a week, so I'll probably be slow to respond...)

**Comment by pllim** [comment]
> Do I need one?

No, you don't. We have a label that would skip that check for you. 😸 

> I'll be effectively unavailable for a week

No worries. We can take over if needed.

**Review by pllim — APPROVED** [review]
Not sure why CI fails. Error looks weird. I'll try to restart tomorrow in case it is temporary server problems.

**Comment by pllim** [comment]
Thank you, @jsoref !

**Pull request merged**